Clinical-Genomics/scout
# Organize pages around each institute

This PR is basically only a portal restructuring without adding new functionalities. So don't worry about the files changed, it's mostly stuff moved from cases blueprints to institutes blueprint. The aim is to make it more user friendly. I was working on #1965 and then I realized that I didn't want to add yet another feature without finding it a proper space in the portal.

So I've wrapped up the following pages:
- cases
- causatives
- variant search (ex gene variants)
- users (small new page)
- clinvar submissions
- institute settings 

into one sidebar, and I've made it the landing page when you click on one institute:

![image](https://user-images.githubusercontent.com/28093618/86457683-56259f80-bd24-11ea-87ef-7a4489aa1ffd.png)


In my vision also other pages should be moved there in the future, for instance gene panel creation, and a new page showing all cases that have been submitted to matchmaker (if scout is connected to it).

I think having the information this way is much more user friendly. Case and variants pages are left unchanged.

I've included fix #1972 since I was working on those pages.


**How to test**:
1. This PR might seem huge but it's just reshuffling the already existent pages. 
1. A reviewer should not focus much on the code (it's mostly all old functions) but on the general effect of having the app restructured in this way. And it would be nice to check that all the links and the pages work, of course!
1. Install the branch on scout stage and navigate around

**Expected outcome**:
Compare the PR against scout prod and see the changes and if they make sense.
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by MM, CR
